src/electionguard/voting/encrypter.c

```diff
@@ -182,6 +182,7 @@ Voting_Encrypter_encrypt_ballot(Voting_Encrypter encrypter,
 
     struct Voting_Encrypter_encrypt_ballot_r balotR;
     balotR.status = VOTING_ENCRYPTER_SUCCESS;
+    uint32_t selected_count;
```

Comment:
You'lll want to revert this file. The `selected_count` was removed in the latest merge in PR #49.